docs/development_suite/webhooks-and-events/events.mdx

```diff
@@ -91,79 +97,697 @@ The event is triggered as soon as the service's repository is successfully creat
+  "payload": {
+    "tenantId": "my-company-id",
+    "projectId": "0000000000000001",
+    "repositoryURL": "http://gihub.com/my-repository",
+    "serviceName": "my-service",
+    "templateId": "0000000000000002",
+    "service": {...}
+  }
+}
+```
+
+</TabItem>
+<TabItem value="json-schema" label="JSON Schema">
+
+```json
+{
+  "additionalProperties": false,
+  "properties": {
+    "eventName": {
+      "const": "service_created",
+      "type": "string"
+    },
+    "eventTimestamp": {
+      "type": "number"
+    },
     "payload": {
-      "tenantId": "my-company-id",
-      "projectId": "0000000000000001",
-      "repositoryURL": "http://gihub.com/my-repository",
-      "serviceName": "my-service",
-      "templateId": "0000000000000002"
+      "additionalProperties": false,
+      "properties": {
+        "tenantId": {
+          "type": "string",
+          "description": "The identifier of the Company the service belongs to"
+        },
+        "projectId": {
+          "type": "string",
+          "description": "The identifier of the Project the service belongs to"
+        },
+        "repositoryURL": {
+          "type": "string",
+          "description": "The URL of the repository that hosts the service code"
+        },
+        "serviceName": {
+          "type": "string",
+          "description": "The name of the service inside the project configuration"
+        },
+        "templateId": {
+          "type": "string",
+          "description": "The identifier of the template in the Console marketplace from which the service was created"
+        },
+        "service": {...}
+      },
+      "type": "object"
     }
+  },
+  "required": ["eventName", "eventTimestamp", "payload"],
+  "type": "object"
 }
 ```
 
 </TabItem>
-<TabItem value="json-schema" label="JSON Schema">
+</Tabs>
+
+## Entities
+
+### Project
+
+<Tabs>
+<TabItem value="project-table" label="Entity">
+
+|        Property        |   Type   |                                         Description                                          |
+| ---------------------- | -------- | -------------------------------------------------------------------------------------------- |
+| `_id`                  | `string` | The unique idetifier of project created                                                      |
+| `configurationGitPath` | `string` | The path where project persist in git repository                                             |
+| `defaultBranch`        | `string` | The default branch of git repository                                                         |
+| `description`          | `string` | The project description                                                                      |
+| `environments`         | `array`  | The project environments, where project can be released                                      |
+| `hostnames`            | `object` |                                                                                              |
```

Comment:
any suggestion?

Reply:
I write below: We can remove this info, also from the project schema and event. It is a deprecated and unused field

Reply:
removed